Lib/fontTools/ttLib/tables/_k_e_r_n.py

```diff
@@ -48,6 +48,16 @@ def decompile(self, data, ttFont):
 				# This "version" is always 0 so we ignore it here
 				_, length, subtableFormat, coverage = struct.unpack(
 					">HHBB", data[:6])
+				if nTables == 1 and subtableFormat == 0:
+					# The "length" value is ignored since some fonts
+					# (like OpenSans and Calibri) have a subtable larger than
+					# its value.
+					nPairs,  = struct.unpack(">H", data[6:8])
+					log.warning(
+						"'kern' subtable longer than defined: "
+						"%d bytes instead of %d bytes",
+						(nPairs * 6) + 14, length)
```

Comment:
@moyogo you only want to print this warning when the `length` stored in the kern subtable is actually different from the one computed from `(nPairs * 6) + 14`. Here it is being logged all the time, whenever nTables == 1 and subtableFormat == 0 (i.e. 99 % of kern tables).